fifteen_min/src/viewer.rs

```diff
@@ -271,13 +271,14 @@ fn build_panel(ctx: &mut EventCtx, app: &App, start: &Building, isochrone: &Isoc
         ctx.style().btn_close_widget(ctx),
     ]));
 
-    rows.push(Widget::row(vec![
-        "Map:".draw_text(ctx),
-        ctx.style()
-            .btn_outline_light_popup(nice_map_name(app.map.get_name()))
-            .hotkey(lctrl(Key::L))
-            .build_widget(ctx, "change map"),
-    ]));
+    rows.push(Widget::row(vec![ctx
```

Comment:
We shouldn't need a `Widget::row` as a wrapper anymore. If you ever need to wrap a single widget to make layouting work, there's a `.container()` method on `Widget`

Reply:
good catch, fixed!